packages/components/package.json

```diff
@@ -42,10 +42,14 @@
     "@storybook/addon-essentials": "^6.4.13",
     "@storybook/addon-links": "^6.4.13",
     "@storybook/react": "^6.4.13",
+    "@testing-library/jest-dom": "^5.16.1",
```

Comment:
do we need this?

Reply:
I am using it to test the textarea component